aggregators/1.0/modules/ROOT/pages/aggregators-sync-async.adoc

```diff
@@ -0,0 +1,35 @@
+= Synchronous Versus Asynchronous Aggregations
+
+An aggregation is complete based on a new element added to the list, as when a max size is specified, or because some timeout or time period is complete. This is important because the type of aggregation determines which chain of components to execute with the list of elements.
+There are two kinds of triggers for an aggregation completion: synchronous and asynchronous.
```

Comment:
We just said that there are two conditions that complete an aggregation: adding a new element or timing out.
Why separate this afterward by calling them triggers and giving them different names?

"An aggregation is considered complete when a new element is added to the elements list (synchronous aggregation) or when a specified timeout is reached or a specified time period is complete (asynchronous aggregation). The type of aggregation determines which chain of components are executed."  ???